frontend/src/components/Notifier/Notifier.jsx

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 import { ToastContainer, toast } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 
-const notify = (message) => toast(message);
+const notify = (message, id, autoClose) => toast(message, {toastId: id || undefined, autoClose: autoClose !== undefined ? autoClose : 3000});
```

Comment:
can you please get rid of `utils/notification` and make this function also accept type param, whiich can be: 'error', 'success', 'info', 'warning', 'default' and based on that render approprietly styled toast? Because currently it seems to show it in default style. 
so it will look like that: `toast[type](message, {....}`

please reffer to https://fkhadra.github.io/react-toastify/introduction/ so you can see the exact API